README.md

```diff
@@ -13,7 +13,7 @@ The Sequelize Command Line Interface (CLI)
 Install CLI globally with
 
 ```bash
-$ npm install -g sequelize-cli
+$ npm install -g sequelize sequelize-cli
```

Comment:
Globally installed cli should be able to pick locally installed `sequelize`

Reply:
Sure, but that's not the point - the point is that a new users (like me) who just want the cli and are getting started will install the cli and it will fail because sequelize isn't installed. So either we update the docs, or we update the deps (which gets more interesting because of the local/global thing).